tutorials/learnpython.org/fr/Welcome.md

```diff
@@ -1,25 +1,65 @@
-Ce site est généreusement soutenu par [DataCamp](https://datacamp.pxf.io/c/67577/1012793/13294?sharedId=learnpython.org). DataCamp offre des [Tutoriels Python](https://datacamp.pxf.io/c/67577/1012793/13294?sharedId=learnpython.org) en ligne interactifs pour la Science des Données. Rejoignez **575&nbsp;000** autres apprenants et commencez à apprendre Python pour la Science des Données aujourd'hui&nbsp;!
-
 # Bienvenue
 
-Bienvenue dans le tutoriel Python interactif LearnPython.org.
+Bienvenue sur le tutoriel interactif de Python de LearnPython.org.
+
+Que vous soyez un programmeur expérimenté ou non, ce site Web est destiné à tous ceux qui souhaitent apprendre le langage de programmation Python.<br>
+
+Vous êtes invité à rejoindre notre groupe sur <a href="http://www.facebook.com/groups/180708015327157/">Facebook</a> pour poser des questions, participer à des discussions et vous tenir informé des mises à jour.
+
+Après avoir terminé les tutoriels, vous pouvez obtenir une certification sur [LearnX](https://www.learnx.org) et ajouter votre certificat à votre profil LinkedIn.
+
+Cliquez simplement sur le chapitre par lequel vous souhaitez commencer et suivez les instructions. Bonne chance !<br>
+
+### Apprenez les bases
+
+- [[Bonjour, le monde !]]
```

Comment:
where are the real links?

Reply:
The remote Welcome.md seems outdated. I’ll push the corrected version with proper links.